gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Mon Dec 17 16:21:35 GMT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
```

Comment:
I think you need some publishing updates with this bump?
see an example of how its done on 5 here:
https://github.com/hmcts/properties-volume-spring-boot-starter/blob/master/build.gradle#L100

Reply:
skimmed through. hope i did not miss a thing. publish local though did not work comparing to linked example ^. but `Main(...)` worked